src/app/common/messagelist.ts

```diff
@@ -89,7 +97,12 @@ export class MessageList extends MessageDisplay {
         name: 'Subject',
         sortColumn: null,
         getValue: (rowIndex: number): string => this.getRow(rowIndex).subject,
-        draggable: true
+        draggable: true,
+        getContentPreviewText: (rowIndex): string => {
+          const ret = this.getRow(rowIndex).plaintext;
+          return ret ? ret.trim() : '';
+        },
+        // tooltipText: 'Tip: Drag subject to a folder to move message(s)'
```

Comment:
Should this be uncommented?

Reply:
It was there when I started.. dunno why its commented ;)

Reply:
I suspect ti's been commented out because a similar message is shown in the snackbar when using Runbox 7 for the first time. :)